pkg/parsers/kernel/kernel_unix.go

```diff
@@ -0,0 +1,34 @@
+// +build linux freebsd
```

Comment:
@thaJeztah thank you for pointing this out!
@allencloud could you please change the build directive to be "// +build linux freebsd solaris" ?